### PR TITLE
Avoid getting default install disk when user assigns one in os instal…

### PR DESCRIPTION
**Pull request by iceiilin**
Provide more flexibility for user's configuration. In case the default satadom os installation failed for some reason, users can assign a disk by their own.

### CODE REVIEW

**Comment by heckj** [comment]
I don't mean to make this harder under a deadline, but the business logic you slipped in there (with good reason) would be perfectly suited to a unit test to verify the inputs and outputs of that job all functioned exactly as expected. Would you please consider adding a unit test to verify that the disk is set to an empty string when appropriate based on your logic?

File: lib/jobs/install-os.js [hunk 1]
```diff
@@ -191,7 +191,13 @@ function installOsJobFactory(
         var self = this;
         return Promise.resolve()
         .then(function () {
-            return self._getInstallDisk();
+            if (self.options.hasOwnProperty('installDisk') &&
+                self.options.installDisk !== '') {
+                return Promise.resolve();
+            }
+            else {
```
Comment (anhou) [inline lib/jobs/install-os.js]: :+1: just style "} else {" is better

**Comment by anhou** [comment]
:+1:  after as joe suggest refine the unit test

**Comment by yyscamper** [comment]
:+1:

**Comment by iceiilin** [comment]
Unit test added.
